script/testing/collect_mem_info.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/local/bin/python3
```

Comment:
I thought /usr/bin/env python3 was generally more reliable, could be mistaken though